test/system/150-login.bats

```diff
@@ -39,6 +43,17 @@ function setup() {
     basic_setup
 }
 
+function teardown() {
+    while read line; do
```

Comment:
Not all tests create that file. Please test for its existence:
```bash
if [[ -e $SNAME_FILE ]]; then
    [indent the rest]
```

Reply:
Should we also do an `unset $NAME_FILE`?

Reply:
`unset`: no need.